assets/react/v3/shared/components/fields/FormSelectInput.tsx

```diff
@@ -53,21 +52,23 @@ const FormSelectInput = <T,>({
 	hideCaret,
 	listLabel,
 	isClearable = false,
-	showArrowUpDown = false,
 	helpText,
 	removeOptionsMinWidth = false,
 	leftIcon,
 	removeBorder,
 	dataAttribute,
 }: FormSelectInputProps<T>) => {
 	const getInitialValue = useCallback(() => {
-		return options.find((item) => item.value === field.value)?.label || '';
+		return options.find((item) => item.value === field.value);
 	}, [options, field.value]);
+	const hasDescription = useMemo(() => options.some((option) => isDefined(option.description)), [options]);
 
-	const [inputValue, setInputValue] = useState(getInitialValue);
+	const [inputValue, setInputValue] = useState(getInitialValue()?.label);
```

Comment:
If only label is reuqired from the `getInitialValue` function then return the label from it. No need to use `useCallback` here.